Source/WebCore/editing/FrameSelection.cpp

```diff
@@ -75,9 +75,14 @@
 #include "StyleTreeResolver.h"
 #include "TypingCommand.h"
 #include "VisibleUnits.h"
+#include "platform/CaretAnimation.h"
+#include "wtf/Assertions.h"
```

Comment:
These need `<>` instead of `“”`.

Reply:
@darinadler This is just a draft PR for my own sake, so it has a bunch of style errors and stuff and it doesn't need to be reviewed until I make it a non-draft PR